sql/core/src/test/resources/sql-tests/inputs/show-tblproperties.sql

```diff
@@ -6,6 +6,13 @@ SHOW TBLPROPERTIES tbl;
 SHOW TBLPROPERTIES tbl("p1");
 SHOW TBLPROPERTIES tbl("p3");
 
+set spark.sql.legacy.keepCommandOutputSchema=true;
```

Comment:
Can we follow others and test the config in the scala test?

Reply:
> Can we follow other PRs and test the config in the scala test?

Updated, should we merge `test("SPARK-34240 Unify output of SHOW TBLPROPERTIES and pass output attributes properly") ` and `test("show tblproperties for hive table") `

Reply:
yea, we can also remove changes in this file.

Reply:
> yea, we can also remove changes in this file.

Done